### DISCUSSION TITLE
Gluten 1.3.0 Discussion

### DISCUSSION THREAD

**Discussion by weiting-chen** [apache/gluten#8225] [General]
This is a discussion for Gluten v1.3.0.
The plan for Gluten v1.3.0 release will be:

- Dec. End: code freeze -> Postpone the schedule to WW02'25, due to some issues in CK backend, which requires more patches.
- Jan. Begin: v1.3.0-rc0 official release -> Plan in WW03'25

This release will cover major feature update, the version of velox upgrade, and more.
Please feel free to join us to discuss Gluten v1.3.0.

**Comment by zhouyuan** [comment]
Here's the feature list I collected so far:
Core features:
	- 1st CK Backend release
	- Sort merge join support 
	- JVM libhdfs with viewfs/kerberos support
	- RAS 
	- Sort based shuffle
	- Partial project(UDF) support
	- More operators and functions
	- Components version
		○ Spark, celeborn, uniffle 
	- Mix backend refactor
		○ Example
	- CI/nightly package tool 
        - New build process
